RELEASE_CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
 ### [v1.0.0-beta3](https://github.com/provenance-io/provwasm/tree/v1.0.0-beta3)
 
 * Update example tutorial to work with the latest version of Provenance
-* expose MetadataMsgParams to library consumers
\ No newline at end of file
+* expose MetadataMsgParams to library consumers
+* Add example scope create/update message to metadata contract README
```

Comment:
It's weird to me that this `RELEASE_CHANGELOG.md` file exists in `main`. It should really only exist in the specific release branches, e.g. `release/0.16.0` or `release/v1.0.0-beta5`. It'd cut down on the duplication of adding entries to both this and `CHANGELOG.md`. And since `main` isn't actually released, having it here is misleading.

Might be some cleanup for later, though.

Reply:
good call. we're still working through a formal release process for provwasm, but this should be in place right now so i'll make the change